Makefile

```
@@ -6,7 +6,7 @@
 # ------------------------------------------------------------------------------
 
 PACKAGE := xcache
-VERSION := 1.1.0
+VERSION := 1.2.1
```

```suggestion
VERSION := 1.2.0
```
We missed this version change